database/migrations/2019_12_26_134611_add_default_value_for_deleted.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddDefaultValueForDeleted extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        // https://stackoverflow.com/a/42107554/9539
+        DB::getDoctrineSchemaManager()->getDatabasePlatform()->registerDoctrineTypeMapping('enum', 'string');
+        Schema::table('replays', function (Blueprint $table) {
+            $table->integer('deleted')->default(0)->change();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
```

Comment:
While down migrations are nice, I think they are optional in many cases, some up migrations are destructive anyways. So in the future you don't need to bother with them.